docs/2-managed-aks-cluster.md

```diff
@@ -31,6 +31,14 @@ az account show
 
 ```
 
+Create the `generated` directory for storing templated files if you have not done so already.
+
+```bash
+
+mkdir -p generated
```

Comment:
don't think -p is required here

Reply:
I added it so it doesn't result in an error if the user already created the directory in a previous lab. Without the -p, it returns an error if the directory already exists.